include/time_series/internal/specialized_classes.hpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) 2019 Max Planck Gesellschaft
+// Vincent Berenz
+
+#pragma once
+
+#include <condition_variable>
+#include <mutex>
+#include <vector>
+
+#include "shared_memory/array.hpp"
+#include "shared_memory/condition_variable.hpp"
+#include "shared_memory/lock.hpp"
+#include "shared_memory/mutex.hpp"
+
+namespace time_series
+{
+namespace internal
+{
+typedef std::integral_constant<int, 0> SINGLEPROCESS;
+typedef std::integral_constant<int, 1> MULTIPROCESSES;
```

Comment:
Like above, types should use UpperCamelCase.

Reply:
+1